packages/components/alert/src/alert.vue

```diff
@@ -26,12 +26,12 @@
         <template v-if="closable">
           <div
             v-if="closeText"
-            :class="[ns.e('closebtn'), ns.is('customed')]"
+            :class="[ns.e('closeBtn'), ns.is('customed')]"
```

Comment:
How about change class name to `close-btn`
```suggestion
            :class="[ns.e('close-btn'), ns.is('customed')]"
```

Reply:
because `closeBtn` is used elsewhere..

Reply:
ok!